mmv1/third_party/terraform/services/cloudrunv2/resource_cloud_run_v2_job_test.go.erb

```diff
@@ -270,12 +271,15 @@ func testAccCloudRunV2Job_cloudrunv2JobWithDirectVPC(context map[string]interfac
 `, context)
 }
 
-func testAccCloudRunV2Job_cloudrunv2JobWithDirectVPCUpdate(context map[string]interface{}) string {
+func testAccCloudRunV2Job_cloudrunv2JobWithDirectVPCAndNamedBinAuthPolicyUpdate(context map[string]interface{}) string {
 	return acctest.Nprintf(`
   resource "google_cloud_run_v2_job" "default" {
     name     = "%{job_name}"
     location = "us-central1"
-    launch_stage = "BETA"
+    binary_authorization {
+      policy = "projects/%{project}/platforms/cloudRun/policies/test-policy"
```

Comment:
policy is in the wrong format, it looks like it shouldn't include `/policies/` based on the error output below:

```
        Error: Error updating Job "projects/ci-test-project-188019/locations/us-central1/jobs/tf-test-cloudrun-serviceawtp88lvds": googleapi: Error 400: Violation in UpdateJobRequest.job.binary_authorization.policy: must be in form projects/{project}/platforms/cloudRun/{policy-name}.
        Details:
        [
          {
            "@type": "type.googleapis.com/google.rpc.BadRequest",
            "fieldViolations": [
              {
                "description": "must be in form projects/{project}/platforms/cloudRun/{policy-name}.",
                "field": "Violation in UpdateJobRequest.job.binary_authorization.policy"
              }
            ]
          }
        ]
```